eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/FaultTolerantObject.java

```diff
@@ -51,7 +51,7 @@ public void onEvent() {
                         creatingNewInnerObject = true;
 
                         try {
-                            openCallbacks.offer(openCallback);
+                            openCallbacks.add(openCallback);
```

Comment:
Why did these calls all change from 'offer' to 'add'?

Reply:
This is my mistake. I didn't mean to include this change. Will revert  and correct original error.